### PR TITLE
Update DependabotSecretsSelectedRepoIDs type to []int64

### PR DESCRIPTION
**Pull request by jburandt**
According to the docs, the API for setting a repos access to dependabot secrets is expecting an array of integers.

 [https://docs.github.com/en/rest/dependabot/secrets?apiVersion=2022-11-28#set-selected-repositories-for-an-organization-secret](https://docs.github.com/en/rest/dependabot/secrets?apiVersion=2022-11-28#set-selected-repositories-for-an-organization-secret)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @jburandt !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
